hack/make.sh

```diff
@@ -118,8 +118,10 @@ fi
 
 if [ -z "$DOCKER_CLIENTONLY" ]; then
 	DOCKER_BUILDTAGS+=" daemon"
-	if pkg-config libsystemd-journal 2> /dev/null ; then
+	if pkg-config 'libsystemd >= 209' 2> /dev/null ; then
```

Comment:
I'm not sure I follow your meaning.  The compiler's going to call out to pkg-config with everything after the ":" to gather flags, so we could include the version requirement in the .go file, but we'd still need to use build tags to get the right file to compile, since pkg-config errors out if it can't find what it's been asked for.  I can add the requirement to the "#cgo pkg-config" line, though, for consistency's sake.